cloudlink/server.py

```diff
@@ -1,9 +1,9 @@
 import sys
 import logging
-from .websocket_server import WebsocketServer
-from .supporter import supporter
-from .serverRootHandlers import serverRootHandlers
-from .serverInternalHandlers import serverInternalHandlers
+from websocket_server import WebsocketServer
```

Comment:
this got fixed in a later commit, github being dumb and not showing it

Reply:
This breaks importing as a module, which is the intended functionality of CL.

`>>> from cloudlink import Cloudlink`
`>>> cl = Cloudlink()`
`>>> server = cl.server(logs=True)`
`Traceback (most recent call last):`
`  File "<stdin>", line 1, in <module>`
`  File "C:\Users\mikie\Documents\GitHub\package\cloudlink\cloudlink.py", line 21, in server`
`    from .server import server`
`  File "C:\Users\mikie\Documents\GitHub\package\cloudlink\server.py", line 3, in <module>`
`    from websocket_server import WebsocketServer`
`ModuleNotFoundError: No module named 'websocket_server'`

Reply:
Please stop removing these relative imports. I intended CL to be imported entirely as a module from external code, or imported as a package from PyPi.